doc/source/whatsnew/v0.15.0.rst

```diff
@@ -836,7 +833,7 @@ Other notable API changes:
 
   .. code-block:: python
```

Comment:
This indentation is wrong, should be at the top level. And this should be a `.. code-block:: ipython` (ipython instead of python.

See similar examples for the exact format.